commands/verify/verify.go

```diff
@@ -112,6 +119,9 @@ func DoVerifyAccordingConfig() error {
 				logger.Log.Warnf("verify case failure, will continue retry, %v", err)
 				time.Sleep(interval)
 			} else {
+				if !failFast {
```

Comment:
Following your documentation, the fail-fast mode means to stop verifying all cases? But following the code, It's only stop the current case? Am I right?

Reply:
I think the "return err" stop the process, so it won't continue to verify the other cases.

Reply:
You are welcome!